docs/zh_CN/PAIMode.md

```diff
@@ -82,4 +82,16 @@ paiConfig:
 
 如果希望将 Trial 的模型数据等其它输出保存到HDFS中，可在 Trial 代码中使用 `NNI_OUTPUT_DIR` 来自己保存输出文件，NNI SDK会从 Trial 的容器中将 `NNI_OUTPUT_DIR` 中的文件复制到 HDFS 中。
 
-如果在使用 pai 模式时遇到任何问题，请到 [NNI Github](https://github.com/Microsoft/nni) 中创建问题。
\ No newline at end of file
+如果在使用 pai 模式时遇到任何问题，请到 [NNI Github](https://github.com/Microsoft/nni) 中创建问题。
+
+## 版本校验
```

Comment:
I'm actually a little confused, why version check is in the PAIMode.md, this sounds more like a generic feature in NNI.

Reply:
I raised this in chat group, please follow up it..